Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -48,8 +48,11 @@ public class OculusXRSDKDeviceManager : XRSDKDeviceManager
         public override void Initialize()
         {
             base.Initialize();
+#if !UNITY_2020_1_OR_NEWER
             UnityEngine.Debug.Log(@"Detected a potential deployment issue for the Oculus Quest. In order to use hand tracking with the Oculus Quest, download the Oculus Integration Package from the Unity Asset Store and run the Integration tool before deploying.
 The tool can be found under <i>Mixed Reality > Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules</i>");
+            // Add a note about abandoning Oculus XRSDK if on 2020+, use open xr instead please
```

Comment:
Is this a TODO, or is this the note?

Reply:
It's a stray comment that's no longer needed. Removing in a followup